parquet/src/main/java/org/apache/iceberg/parquet/ParquetWriter.java

```diff
@@ -195,11 +196,12 @@ private void startRowGroup() {
     this.nextCheckRecordCount = Math.min(Math.max(recordCount / 2, 100), 10000);
     this.recordCount = 0;
 
-    PageWriteStore pageStore = pageStoreCtorParquet.newInstance(
+    ColumnChunkPageWriteStore pageStore = pageStoreCtorParquet.newInstance(
```

Comment:
Why are there write-side changes in this PR?

Reply:
Hi, I have split this into 2 PRs:
Core: Support writing parquet bloom filter #2642
Core: Support reading parquet bloom filter #2643